packages/flame_3d/pubspec.yaml

```diff
@@ -8,14 +8,15 @@ funding:
   - https://patreon.com/bluefireoss
 
 environment:
-  sdk: '>=3.3.0-279.2.beta <4.0.0'
-  flutter: ">=3.13.0"
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: ">=3.19.0"
```

Comment:
not sure why this was changed but seems to be an "upgrade"

Reply:
This version is too low to work with the 3d stuff afaik

Reply:
But since it was even lower before I guess it doesn't matter :smile:

Reply:
it def works, just need to follow @wolfenrain 's tutorial

Reply:
But you're not on 3.19 locally, you're on some commit off master right?

Reply:
Yea this should be fine, the constraint is good enough to allow for master commits